m3/reporter.go

```diff
@@ -226,10 +227,12 @@ func NewReporter(opts Options) (Reporter, error) {
 	}
 
 	for k, v := range tagm {
-		tags = append(tags, m3thrift.MetricTag{
-			Name:  k,
-			Value: v,
-		})
+		tags = append(
+			tags, m3thrift.MetricTag{
+				Name:  k,
+				Value: v,
+			},
+		)
```

Comment:
Let's revert this and similar formatting changes - it was more readable before, and we shouldn't split a function call over multiple lines without also putting each parameter on its own line.

Reply:
done